app/models/product.rb

```diff
@@ -1,5 +1,14 @@
 class Product < ApplicationRecord
-  belongs_to :category
+  scope :ordered_by_price, ->{order(price: :asc)}
   has_many :rates, dependent: :destroy
   has_many :order_details, dependent: :destroy
+  belongs_to :category, foreign_key: "categories_id"
```

Comment:
```suggestion
  belongs_to :category
```
chị nghĩ chỉ cần ri thôi nhỉ
mà sao field là là categries_id nhri, đúng thì là `category_id` chứ

Reply:
**categories_id** là khóa ngoại trỏ trực tiếp tới chính nó category_id dùng cho multicategory ấy chị

Reply:
Nếu trường `categories_id` là do em đặt thì nên đổi tên thành `category_id` nhé vì mỗi bản ghi  chỉ lưu 1 id category nên ko phải số nhiều em nhé